src/css/styles.scss

```diff
@@ -1,6 +1,42 @@
 @import 'variables';
 @import 'mixins.scss';
 
+@mixin inputFields {
+  margin: auto;
+  color: $header_color;
+  margin-top: 10px;
+  text-align: left;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+@mixin inputBoxes {
+  width: 100%;
+  font-size: 1.2em;
```

Comment:
not sure if it would work, but we do have font-size variables that might be able to be used here.